### PR TITLE
Chore: release notes 2021.03

### LINKED ISSUE #5526: release note 2021.03
**Describe the issue. What is the expected and unexpected behavior?**
Update release notes for 2021.03

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #5526

### CODE REVIEW

**Comment by redallen** [comment]
Still waiting on https://github.com/patternfly/patternfly/pull/3909 and then a core bump.

**Comment by nicolethoen** [comment]
@tlabaj could we also regenerate the example screen shots as part of this release-notes PR?

**Comment by redallen** [comment]
After the core bump the commands to update the screenshots are:
1. `yarn build:docs && yarn serve:docs`
2. (new terminal) `yarn workspace @patternfly/react-docs screenshots` (you can add `"screenshots": "yarn workspace @patternfly/react-docs screenshots"` to the root `package.json` to just have to run `yarn screenshots` in the future)

**Comment by redallen** [comment]
We'll need to fix https://app.circleci.com/pipelines/github/patternfly/patternfly-react/11541/workflows/21c9cda3-a75d-46a9-8250-57c06de19d60/jobs/54794, update the version numbers here, and then finally merge it.

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,61 @@ id: Release notes
...
 
+## 2021.03 release notes (2021-03-10)
+- [@patternfly/react-catalog-view-extension@4.10.13](https://www.npmjs.com/package/@patternfly/react-catalog-view-extension/v/4.10.13)
+- [@patternfly/react-charts@6.14.6](https://www.npmjs.com/package/@patternfly/react-charts/v/6.14.6)
+- [@patternfly/react-code-editor@4.2.2](https://www.npmjs.com/package/@patternfly/react-code-editor/v/4.2.2)
+- [@patternfly/react-core@4.101.3](https://www.npmjs.com/package/@patternfly/react-core/v/4.101.3)
+- [@patternfly/react-icons@4.9.5](https://www.npmjs.com/package/@patternfly/react-icons/v/4.9.5)
+- [@patternfly/react-inline-edit-extension@4.6.48](https://www.npmjs.com/package/@patternfly/react-inline-edit-extension/v/4.6.48)
+- [@patternfly/react-styles@4.8.5](https://www.npmjs.com/package/@patternfly/react-styles/v/4.8.5)
+- [@patternfly/react-table@4.23.13]4](https://www.npmjs.com/package/@patternfly/react-table/v/4.23.14)
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:14]: `13]4]` :(

**Review by redallen — CHANGES_REQUESTED** [review]
syntax

**Review by redallen — APPROVED** [review]
LGTM!

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**